Netconf-notif-v12-examples-v17Sept2018/netconf-notif-4-modify-datastore.xml

```diff
@@ -4,11 +4,11 @@
        xmlns="urn:ietf:params:xml:ns:yang:ietf-subscribed-notifications"
        xmlns:yp="urn:ietf:params:xml:ns:yang:ietf-yang-push">
     <id>23</id>
-    <yp:datastore-xpath-filter xmlns="http://example.com/datastore"> 
-        /interfaces-state/interface/oper-status
+    <yp:datastore-xpath-filter xmlns:ds="http://example.com/datastore"> 

```

Comment:
explict NS here is **required**. Doesn't have to be what is hsown here, though. XPath needs to be consustent.